iis-common/src/main/java/eu/dnetlib/iis/common/report/PigCounterValueResolver.java

```diff
@@ -0,0 +1,65 @@
+package eu.dnetlib.iis.common.report;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.commons.lang3.StringUtils;
+
+import eu.dnetlib.iis.common.counter.PigCounters;
+
+/**
+ * Resolver of values with pig counters placeholder.
+ * 
+ * @author madryk
+ */
+public class PigCounterValueResolver {
```

Comment:
This class looks weird. I guess it is focused on too many things. One thing is to parse some string into an alias and counter name and the other is to get from pigCounters a counter that is assigned the given alias and name.